test/groovy/util/JenkinsShellCallRule.groovy

```diff
@@ -79,10 +79,8 @@ class JenkinsShellCallRule implements TestRule {
                         for (Command failingCommand: failingCommands){
                             if(failingCommand.type == Type.REGEX && unifiedScript =~ failingCommand.script) {
                                 throw new Exception("Script execution failed!")
-                                break
```

Comment:
Revoving the breaks here is very reasonable. But what about having a focused PR for this?